Specs/Core/VideoSynchronizerSpec.js

```diff
@@ -160,28 +160,37 @@ defineSuite([
         var epoch = JulianDate.fromIso8601('2015-11-01T00:00:00Z');
         var clock = new Clock();
 
-        var element = loadVideo();
+        // Since Chrome doesn't allow video playback without user
+        // interaction, we use a mock element.
+        var element = jasmine.createSpyObj('MockVideoElement', ['addEventListener', 'removeEventListener']);
+        element.paused = false;
+        element.play = function() {
+            this.paused = false;
+        };
+        element.pause = function() {
+            this.paused = true;
+        };
+        spyOn(element, 'play').and.callThrough();
```

Comment:
You can create play/pause in `createSpyObj` and then use `callFake` here to provide implementation instead of also defining play/pause above.